client/src/stores/user/reducers.js

```diff
@@ -0,0 +1,28 @@
+import { LOAD_USER_DATA, UPDATE_USER_DATA } from "./actions";
+
+const initialState = {
```

Comment:
Aside: One of Fabrice's comments was that we could consider adding a profile picture for each user, to make the app more colorful. What do you think of that suggestion?

Reply:
I think that is a good idea, but I was wondering if that is something we would import from their fb/google profile or would we allow them to choose from some pre-selected images we provide?

If they're allowed to pick from some pre-vetted options, storing the information doesn't seem very difficult. However, if we get it from their fb/google profile, I'm not sure how would we store this information in the user state. Do you have any insight about that?

Reply:
I think getting it from FB/Google is the easiest for now. We don't currently store the profile image in the DB, but I believe we should be able to get it from the respective website. For Facebook, we could make the following query to get the URL of a profile pic:

```json
/v9.0/me/picture?type=large
```